### DISCUSSION TITLE
GridApps D Python installation

### DISCUSSION THREAD

**Discussion by MaximeBaudette** [GRIDAPPSD/gridappsd-forum#97] [API] [answered]
I'm trying to install GridApps D python API, but I'm getting a "ConnectFailedException:" when trying to instanciate the GridApps object. I've seen in your build logs that you have a similar error. 

Could you point me to a tag of the latest stable install so I can pip install a working version?

**Comment by poorva1209** [comment, marked as answer]
@tonya1 @craig8 I think this is happening because gridappsd:develop container does not have the latest gridappsd-python with token-auth updates.  Can you manually push the latest gridappsd-python to GridAPPS-D develop container to resolve this issue? 

@MaximeBaudette Meanwhile, please use './run.sh -t v2020.11.0" for the last stable release after executing "./stop.sh -c" to remove all the containers and data for the currently installed develop tag.

**Reply by MaximeBaudette** [reply]
I'm having trouble launching a certain tag, do I need to update the repository to a certain commit? I should be up to date on the develop branch

![Screen Shot 2021-02-12 at 14 05 39](https://user-images.githubusercontent.com/15156063/107827336-6b8f8d00-6d3b-11eb-8045-b08e242fcced.png)

**Reply by poorva1209** [reply]
I just tried that tag on my local VM and ran a simulation. It is working fine for me. Maybe you are having some network issues. Try again after some time and update here.

**Reply by MaximeBaudette** [reply]
Ok this got resolved by rebooting Docker. I managed to launch gridapps with the specific tag, but still have connection issues from the python API

**Reply by poorva1209** [reply]
What version of gridappsd-python are you using? If it is develop then change it to following release: https://github.com/GRIDAPPSD/gridappsd-python/tree/releases/2020.11.0

**Reply by MaximeBaudette** [reply]
I think I'm on the develop branch of the python API, I'll try that.

**Reply by MaximeBaudette** [reply]
Thanks, it's working now :-)
